libs/client/cornflow_client/core/experiment.py

```diff
@@ -81,3 +83,59 @@ def schema_checks(self) -> dict:
             the method ExperimentCore.check_solution()
         """
         raise NotImplementedError()
+
+    @staticmethod
+    def get_solver_config(config, lib="pyomo", default_solver="cbc", remove_unknown=False):
+        """
+        Format the configuration used to solve the problem.
+        Solver configuration can either be directly in config using cornflow mapping name
+           or in a config["solver_config"] using the solver names.
+        Example:
+            config = {
+                "solver":"milp.cbc",
+                "time_limit":60,
+                "rel_gap":0.1,
+                "solver_config":{"heur":1, "pumpC":0}
+            }
+
+        :param config: dict config argument of the solver method
+        :param lib: str library used to create the model (pulp or pyomo)
+        :param default_solver: str default solver to use if none is present inf config.
+        :return: the solver name and the config dict.
+        """
+        mapping = PARAMETER_SOLVER_TRANSLATING_MAPPING
+        solver = config.get("solver", None)
+
+        if solver is None:
+            message = f"No solver found in config. Default solver {default_solver} will be used"
+            print(message)
+            solver = default_solver
+
+        if "." in solver:
+            solver = solver.split(".")[1]
+
+        solver = SOLVER_CONVERTER.get(solver)
+        if solver is None:
+            print(
+                "The solver doesn't correspond to any solver in the parameters mapping. "
+                f"Default solver {default_solver} will be used."
+            )
+            solver = default_solver
+
+        if remove_unknown:
```

Comment:
I do not really understand the use of remove_unknown

Reply:
If it is True, the method removes the parameters that are not in PARAMETER_SOLVER_TRANSLATING_MAPPING from the config. Otherwise it just doesn't translate it